core/util/src/main/scala/net/liftweb/util/Props.scala

```diff
@@ -328,5 +388,8 @@ object Props extends Logger {
         Map()
     }
   }
+
+  private var sources:List[PropProvider] = List(props)
+  private var interpolators:List[PropProvider] = Nil
```

Comment:
These should either both be volatile, or we should lock down source/interpolator management at some point.